packages/react-components/src/components/TextInput/TextInput.jsx

```diff
@@ -70,14 +81,14 @@ class TextInput extends React.Component {
     // Calculate max characters and display '(Max. XX characters)' when max is hit.
     if (this.props.field.value) {
       if (this.props.charMax === this.props.field.value.length) {
-        maxCharacters = <small>(Max. {this.props.charMax} characters)</small>;
+        maxCharacters = <small>{i18next.t('max-chars', { length: this.maxlength })}</small>;
```

Comment:
We should include the parentheses here as well.

```suggestion
        maxCharacters = <small>({i18next.t('max-chars', { length: this.maxlength })})</small>;
```